### ISSUE TITLE
Git Gateway Error accessing Admin panel on a brand-new one-click deployment

### ISSUE THREAD

**Issue by dvejmz** [netlify-templates/one-click-hugo-cms#658]
**- Do you want to request a *feature* or report a *bug*?**
Bug.

**- What is the current behavior?**

I get the following error when I try to log in with an existing Netlify Identity user account on a brand-new NetlifyCMS Hugo starter template site.

>Git Gateway Error: Please ask your site administrator to reissue the Git Gateway token.

I cannot proceed to the next screen. The underlying response is a HTTP 401 with the following payload

```
{
  "message": "Bad credentials",
  "documentation_url": "https://docs.github.com/rest"
}
```
![zen-wing-14c835 netlify app_admin_](https://user-images.githubusercontent.com/9487006/153096077-403ee857-5e39-4707-8cb3-7eeb008b2f60.png)

**- If the current behavior is a bug, please provide the steps to reproduce.**

1. Launch a new NetlifyCMS Hugo starter template, leaving all defaults in, i.e. Git Gateway and Netlify Identity are enabled, GitHub API token is set...
2. Accept the email invite that will arrive when the website is finished deploying
3. Sign up on the newly deployed site and navigate to admin page on `/admin`
4. Error described above appears and you're not allowed to proceed further.

**- What is the expected behavior?**

If I attempt to log onto the NetlifyCMS Admin dashboard with an existing Netlify Identity account, I should be allowed to proceed.

**- Please mention your node.js, NPM, Hugo and operating system version.**

It's the Netlify-hosted version pointing [at the HEAD of this repo](https://github.com/netlify-templates/one-click-hugo-cms/commit/79e955be2d36900929b2f75ace3c3d67577ef1bf), so whatever those versions are. This happens both on Chrome and Firefox running on my Linux machine.

**Comment by dvejmz** [comment]
I found a fix for this. I created a new git branch called `master` and pushed it up to the GitHub remote. I then tried to log in again and this time it was successful. I don't know why admin auth via Netlify ID doesn't work when a branch called `master` doesn't exist in the remote, even though `main` is set as the the default branch in git as well as the production brand in Netlify. Maybe some older, hardcoded assumption to check for `master` instead of `main`? Note that I did not even need to deploy from `master` to fix the problem: the fact that the GitHub API call the Netlify Hugo app makes finds a `master` branch as expected is enough for it to work. See screenshot attached of API calls when login is unsuccessful.

![2022-02-11T21:36:10,993931938+00:00](https://user-images.githubusercontent.com/9487006/153674134-ce0f2c4b-9f2e-4513-87cb-0d634221e403.png)

**Comment by Zackyist** [comment]
Thank you so much for the workaround, I was beating my head to the wall with this earlier!

**Comment by jwsy** [comment]
I was able to get this working by adding the `branch: main` attribute for backend obj at the top of **./site/static/admin/config.yml**
```
backend:
  name: git-gateway
  branch: main # add this line
```

**Comment by maxcell** [comment]
Ack! This is a gnarly bug y'all stumbled into! I fully agree with @jwsy's solution for changing the branch to main. I've recently made this change to the HEAD of the project so folks hopefully don't have this happen to them for the future! Thank you so much for reporting it and hope to be able to address any more problems with y'all!

**Comment by maxcell** [comment]
Closed with #674
